PersonDetails.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Person Details</title>
+
+	<style type="text/css">
+	.container{
+		margin: auto;
+	    width: 960px;
+	    border:1px solid;
+	    list-style-type: none;
+	    overflow:hidden;
+	}
+	li{
+		float: left;
+		width: 300px;
+		height: 400px;
+		border: 1px solid;
+		margin-bottom: 47px;
+		
+	}
+	li:nth-child(3n-1) {
+		margin-left: 47px;
+		margin-right: 47px;
+	}
+	li:nth-child(3n-2) {
+		margin: 0 0 0 -40px;
+		
+	}
+	img{
+		width: 200px;
+		height: 200px;
+		
+		padding: 15px;
+	}
+	.inner{
+	   list-style:none;
+	}
+	</style>
+
+</head>
+<body>
+	<h1 align="center">List of Person Details</h1>
+	
+	<ul class="container">
+		<li>
+			<ul class="inner">
```

Comment:
under ul element typically you find li.  In this case you are just defining an unordered list but  the content of it is not a list.  So you can get rid of the ul and replace it with a div